applications_tests/cfd_dem_coupling_3d/dynamic_contact_search.output

```diff
@@ -59,9 +59,9 @@ Finished 100 DEM iterations
 Global continuity equation error: -1.507798244e-18 s^-1
 Max local continuity error: 1.562921799e-11 s^-1
 
-*****************************************************************
-Transient iteration : 3        Time : 0.015    Time step : 0.005    CFL : 1.44013875e-07
-*****************************************************************
+**************************************************************************************
+Transient iteration : 3        Time : 0.015    Time step : 0.005    CFL : 1.44014e-07
```

Comment:
Why did you change the precision?

Reply:
I did not change the precision, just changed the computer.